torch/_export/converter.py

```diff
@@ -89,6 +91,16 @@ def __init__(
 
         self.subgraphs: Dict[str, torch.fx.GraphModule] = {}
 
+        # Populate methods for the standard operators.
+        for k in kind_to_standard_operators.keys():
+            k_list = k.split("::")
+            func_name = "convert_" + "_".join(k_list)
```

Comment:
We should put this in some helper function so that we can consolidate the same logic here with the logic inside of convert_node.